app/src/main/java/fr/free/nrw/commons/notification/NotificationWorkerFragment.java

```diff
@@ -0,0 +1,29 @@
+package fr.free.nrw.commons.notification;
+
+import android.app.Fragment;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+
+import java.util.List;
+
+/**
+ * Created by knightshade on 25/2/18.
+ */
+
+public class NotificationWorkerFragment extends Fragment {
```

Comment:
@knight-shade I am confused why you added `NotificationWorkerFragment`. Can you briefly explain the intent behind it.

Reply:
Sure @maskaravivek , the sole purpose of NotificationWorkerFragment is to retain notifications list (data) initially fetched from the server across orientation change. Its a headless fragment , so after orientation change when the Notification activity is recreated, instead of fetching notification list again from server, it uses the initially fetched notification list which is stored is NotificationWorkerFragment.

Reply:
@maskaravivek I am convinced that this PR is beneficial for us. What do you think?